README.md

```diff
@@ -35,6 +36,15 @@ Experts provide support for decisions but do not act as gatekeepers.
 - **Ownership**: Independency means ownership, because teams have to live with their decisions
 - **Motivation**: Autonomy motivates, because teams are expected to find solutions for their challenges.
 
+## Continuous Delivery
+
+We organize and develop our software as small decoupled services, with named engineers owning the maintenance and development of each service. These services are then integrated and deployed through a unified continuous delivery (CD) pipeline, incorporating automated end-to-end tests verifying functionality in the context of realistic use cases and data.
+
+### Why?
+- **Reliability** Our customers expect all features to work seamlessly together at all times. Testing end-to-end with realistic use cases helps prevent most regressions.
```

Comment:
Love this! Would also save manual cross-team testing especially when we have the right tests in place.